grails-app/jobs/org/pih/warehouse/jobs/SessionlessJob.groovy

```diff
@@ -0,0 +1,8 @@
+package org.pih.warehouse.jobs
+
+class SessionlessJob {
+    // By default this is true on QuartzDisplayJob, which invokes execute()
+    // and if sessionRequired is true, then QuartzDisplayJob tries to do session flush
+    // even if there is no session
+    static sessionRequired = false
```

Comment:
I don't think this needs to be static.

FYI, here are some examples where the property is set in the class.
https://github.com/search?q=sessionRequired+language%3AGroovy+path%3Agrails-app%2Fjobs&type=code

Reply:
No, you're right it does not need to be static